src/components/ProjectView/ImageList.vue

```diff
@@ -1,41 +1,55 @@
 <script setup>
-    import { ref } from 'vue'
-    const props = defineProps(['projectImageObjects', 'selectedImages', 'data'])
+import { ref, onMounted } from "vue";
+import { useStore } from "vuex";
 
-    // v-data-table setup variables
-    let headers = ref([
-        { title: 'Image Name', align: 'start', sortable: true, key: 'basefile_name'},
-        { title: 'Time', align:'start', sortable: true, key: 'time'},
-        { title: 'Object', align:'start', sortable: true, key: 'object'},
-        { title: 'Image', align:'start', sortable: false, key: 'image'}
-    ])
-    let items = ref(props.data)
-    let itemsPerPage = ref(15)
-    let selected = ref([])
+const props = defineProps(["data"]);
+const store = useStore();
+
+// v-data-table setup variables
+let headers = ref([
+  { title: "Image Name", align: "start", sortable: true, key: "basefile_name" },
+  { title: "Time", align: "start", sortable: true, key: "time" },
+  { title: "Object", align: "start", sortable: true, key: "object" },
+  { title: "Image", align: "start", sortable: false, key: "image" },
+]);
+let items = ref(props.data);
+let itemsPerPage = ref(15);
```

Comment:
I think we can change this to be an infinite scroll and add lazy loading. It can be a separate story though : )

Reply:
added to board